messaging/src/main/java/org/axonframework/commandhandling/SimpleCommandBus.java

```diff
@@ -72,6 +75,7 @@ public class SimpleCommandBus implements CommandBus {
             new CopyOnWriteArrayList<>();
     private final CommandCallback<Object, Object> defaultCommandCallback;
     private RollbackConfiguration rollbackConfiguration;
+    private final SpanFactory spanFactory;
 
     /**
      * Instantiate a Builder to be able to create a {@link SimpleCommandBus}.
```

Comment:
Note that the `builder` methods throughout also define the default values. Hence, I'd expect all the infra components where you have added this default field to have a change in the `builder`'s JavaDoc too.